README.md

```diff
@@ -73,6 +73,10 @@ Receive notifications of critical security updates. The sophistication of online
 - [See who's using Mattermost](https://mattermost.com/blog/category/customer-stories/)
 - [Browse over 700 Mattermost integrations](https://integrations.mattermost.com/)
 
+## License
+
+See the [LICENSE file](LICENSE.txt) for license rights and limitations.
```

Comment:
@jasonblais I have changed the text as asked by you also I have shortened the URL to the license agreement file

Reply:
@jasonblais please let me know if the shortened URL will work or I may change it to a full link